scalameta/semantic/shared/src/main/scala/scala/meta/semantic/Symbol.scala

```diff
@@ -35,10 +36,10 @@ object Symbol {
     override def fullName: Ref = unsupported(this, "fullName")
   }
 
-  @leaf class Local(position: Position) extends Symbol {
```

Comment:
`Input.LabeledString` would be nice, but you need `contents` to create it, and we don't have that when parsing a local symbol.